src/transaction/TransactionController.ts

```diff
@@ -544,7 +574,7 @@ export class TransactionController extends BaseController<TransactionConfig, Tra
     await this.sign(ethTransaction, transactionMeta.transaction.from);
     const rawTransaction = bufferToHex(ethTransaction.serialize());
     await query(this.ethQuery, 'sendRawTransaction', [rawTransaction]);
-    transactionMeta.status = 'cancelled';
+    transactionMeta.status = TransactionStatus.cancelled;
```

Comment:
what are your thoughts on de-structuring these?

Reply:
Do you mean like this?

```suggestion
    const { cancelled } = TransactionStatus
    transactionMeta.status = cancelled;
```

That seems... longer and more complicated to me? Maybe I'm missing something. I try to avoid having extraneous local variables unless it improves readability, and I don't think it would in this case.